packaging/installer/README.md

```diff
@@ -165,3 +165,23 @@ the community helps fix any bugs that might have been introduced in previous rel
 -   Protect yourself from the rare instance when major bugs slip through our testing and negatively affect a Netdata
     installation
 -   Retain more control over the Netdata version you use
+
+## Installation notes
+
+We are tracking a few ongoing issues related to installation and packaging.
+
+**Multiple versions of SSL**: We've received reports from the community about issues with running the
+`kickstart.sh` script on systems that have both a distribution-installed version of OpenSSL and a manually-installed
+local version. The Agent's installer cannot handle both, and so you must remove one or the other to install the Agent.
+
+**LibreSSL**: The Agent installer is only compatible with OpenSSL. Critical functions do not work on systems with
+LibreSSL installed as a replacement for OpenSSL.
```

Comment:
Actually, the build fails completely if you try to use LibreSSL instead of OpenSSL. They use the same naming for libraries and headers but aren't API compatible.

Reply:
@Ferroin I am working on a new PR with TLS right now, I will bring a fix for this on it.

Reply:
@Ferroin Done.

@thiagoftsm We can always edit this text again when your PR is ready.